shell/edit/management.cattle.io.fleetworkspace.vue

```diff
@@ -39,8 +39,11 @@ export default {
     if (this.$store.getters['management/schemaFor']( FLEET.CLUSTER )) {
       this.fleetClusters = await this.$store.dispatch('management/findAll', { type: FLEET.CLUSTER });
     }
+    const repoRestrictionSchema = this.$store.getters['management/schemaFor']( FLEET.GIT_REPO_RESTRICTION );
 
-    if (this.$store.getters['management/schemaFor']( FLEET.GIT_REPO_RESTRICTION )) {
+    this.canCreateRepoRestrictions = repoRestrictionSchema && repoRestrictionSchema.resourceMethods?.includes('PUT');
+    this.canViewRepoRestrictions = repoRestrictionSchema && repoRestrictionSchema.resourceMethods?.includes('GET');
```

Comment:
Yikes! I think we should be careful about trying to work out permissions based off something other than schema resourceMethods. We can talk about what to do about #8468 over slack. I've pulled the other fleet issue into a separate PR https://github.com/rancher/dashboard/pull/8500 because it seems more...self-contained, still worth getting in 2.7.2